lib/src/easy_localization_app.dart

```diff
@@ -95,7 +96,10 @@ class _EasyLocalizationState extends State<EasyLocalization> {
 
   @override
   void initState() {
-    _locale = _EasyLocalizationLocale(widget.fallbackLocale);
+    _locale =
+        _EasyLocalizationLocale(widget.fallbackLocale, this.supportedLocales);
+    _locale.initSavedAppLocale();
```

Comment:
@aissat by doing this here you will cause a flicker if the user has selected a locale the previous the app was open that is different from the device locale. That is why i was doing this ensureInitialized call in main

Reply:
By flicker i mean that text will be initialy in the device locale and then change to the previous saved locale

Reply:
I don't noticed this, everything works well but really great **note** it true

> text will be initialy in the device locale and then change to the previous saved locale